src/main/webapp/app/overview/course-lectures/course-lecture-details.module.ts

```diff
@@ -0,0 +1,44 @@
+import { ArtemisSharedModule } from 'app/shared/shared.module';
+import { ArtemisSharedComponentModule } from 'app/shared/components/shared-component.module';
+import { ArtemisSharedPipesModule } from 'app/shared/pipes/shared-pipes.module';
+import { MomentModule } from 'ngx-moment';
+import { ArtemisLectureUnitsModule } from 'app/overview/course-lectures/lecture-units.module';
+import { CourseLectureDetailsComponent } from 'app/overview/course-lectures/course-lecture-details.component';
+import { NgModule } from '@angular/core';
+import { ArtemisLearningGoalsModule } from 'app/course/learning-goals/learning-goal.module';
+import { UserRouteAccessService } from 'app/core/auth/user-route-access-service';
+import { Authority } from 'app/shared/constants/authority.constants';
+import { RouterModule, Routes } from '@angular/router';
+
+const routes: Routes = [
+    {
+        path: '',
+        component: CourseLectureDetailsComponent,
+        data: {
+            authorities: [Authority.USER],
+            pageTitle: 'overview.lectures',
+        },
+        canActivate: [UserRouteAccessService],
+        children: [
+            {
+                path: '',
+                pathMatch: 'full',
+                loadChildren: () => import('../../overview/student-questions/student-questions.module').then((m) => m.ArtemisStudentQuestionsModule),
```

Comment:
This doesn't look correct. Before we used this to lazily load the `ArtemisStudentQuestionsModule` once you accessed the lecture details component. But now that component is lazily loaded itself, which means we lazy load twice here? (Lazy load `ArtemisCourseLectureDetailsModule` and then immediately `ArtemisStudentQuestionsModule` because they match the same path?)

Reply:
This is the only combination that got it to work